File: iFlat/src/main/java/com/iflat/system/bean/Operating.java
package com.iflat.system.bean;

/**
 * Created by tyriv on 2015/9/20.
 */
public class Operating {
    //id
    private String opId;
    //操作
    private String operating;
    //别名
    private String alias;
    //方法
    private String method;
    //排序
    private String sequence;
    //nameSpace
    private String nameSpace;
    //模块名
    private String moduleName;

    public String getOpId() {
        return opId;
    }

    public void setOpId(String opId) {
        this.opId = opId;
    }

    public String getSequence() {
        return sequence;
    }

    public void setSequence(String sequence) {
        this.sequence = sequence;
    }

    public String getOperating() {
        return operating;
    }

    public void setOperating(String operating) {
        this.operating = operating;
    }

    public String getAlias() {
        return alias;
    }

    public void setAlias(String alias) {
        this.alias = alias;
    }

    public String getMethod() {
        return method;
    }

    public void setMethod(String method) {
        this.method = method;
    }

    public String getNameSpace() {
        return nameSpace;
    }

    public void setNameSpace(String nameSpace) {
        this.nameSpace = nameSpace;
    }

    public String getModuleName() {
        return moduleName;
    }

    public void setModuleName(String moduleName) {
        this.moduleName = moduleName;
    }
}

File: iFlat/web/app/model/system/Operating.js
Ext.define('iFlat.model.system.Operating', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'operating.opId', mapping: 'opId', type: 'string'},
        {name: 'operating.operating', mapping: 'operating', type: 'string'},
        {name: 'operating.alias', mapping: 'alias', type: 'string'},
        {name: 'operating.method', mapping: 'method', type: 'string'},
        {name: 'operating.sequence', mapping: 'sequence', type: 'string'},
        {name: 'operating.nameSpace', mapping: 'nameSpace', type: 'string'},
        {name: 'operating.moduleName', mapping: 'moduleName', type: 'string'},
    ]
});